modules/kubernetes-addons/helm-addon/locals.tf

```diff
@@ -1,10 +0,0 @@
-terraform {
-  experiments = [module_variable_optional_attrs]
```

Comment:
Locals can go, but I believe this experiments flag needs to stay due to the use of `option()` in variable type definitions https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/cd9a391c5600c1d3c83cfcf7bea76bcb969c3ca7/modules/kubernetes-addons/helm-addon/variables.tf#L30